.github/workflows/visualize.yml

```diff
@@ -0,0 +1,177 @@
+name: visualize
+on:
+ push:
+  branches:
+    - main
+ workflow_dispatch:
+
+jobs:
+  run-must:
+    runs-on: ubuntu-latest
+    env:
+      MPI_CORRECTNESS_BM_DIR: ${{ github.workspace }}
+      MPI_CORRECTNESS_BM_EXPERIMENT_DIR: ${{ github.workspace }}/experiment
+    steps:
+      - uses: actions/checkout@v2
+# prerequisites for must
+      - run : sudo apt-get update && sudo apt-get install -y cmake git libxml2-dev openmpi-bin libopenmpi-dev
+      - run : sudo pip3 install numpy pandas lxml html5lib BeautifulSoup4
+# install MUST
+#      - name: Cache MUST
+#        id: cache-must
+#        uses: actions/cache@v2
+#        with:
+#          path: MUST_install
+#          key: ${{ runner.os }}-must-1.7.0
+      
+      - name: Build MUST
+#        if: steps.cache-must.outputs.cache-hit != 'true'
+        run: |
+              bash ./scripts/MUST/install_tool.sh
+# setup env or running
+      - run: mkdir experiment
+      - name: Run CorrBench
+        run: |
+           export PATH=$PATH:${GITHUB_WORKSPACE}/MUST_install/bin && \
+           export LIBRARY_PATH=$LIBRARY_PATH:${GITHUB_WORKSPACE}/MUST_install/lib && \
+           export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:${GITHUB_WORKSPACE}/MUST_install/lib && \
+           export CPATH=$CPATH:${GITHUB_WORKSPACE}/MUST_install/include && \
+           bash ${GITHUB_WORKSPACE}/scripts/tool_runner_CI.sh MUST
+      - uses: actions/upload-artifact@v2
+        with:
+          name: results_MUST
+          path: ${{ github.workspace }}/experiment/MUST/results_MUST.json
+
+
+
+
+  run-parcoach:
+    runs-on: ubuntu-latest
+    env:
+      MPI_CORRECTNESS_BM_DIR: ${{ github.workspace }}
+      MPI_CORRECTNESS_BM_EXPERIMENT_DIR: ${{ github.workspace }}/experiment
+    steps:
+      - uses: actions/checkout@v2
+# prerequisites
+      - run : sudo apt-get update && sudo apt-get install -y cmake git libopenmpi-dev clang-9 llvm-9 llvm-9-dev
+# install PARCOACH
+# building parcoach is quite fast anyway
+#      - name: Cache PARCOACH
+#        id: cache-parcoach
+#        uses: actions/cache@v2
+#        with:
+#          path: parcoach
+#          key: ${{ runner.os }}-cache-parcoach-master
+      #TODO BUILD
+      - name: Build PARCOACH
+#        if: steps.cache-parcoach.outputs.cache-hit != 'true'
+        run: |
+           bash ./scripts/PARCOACH/install_tool.sh
+#TODO provide a container with the installation of MUST
```

Comment:
Should we have this TODO as an issue in the github issue tracker?

Reply:
If we want container with the tools installed, we should add it to the issues - I'll remove it from the file